app/model.py

```diff
@@ -1012,52 +1018,50 @@ def create_id():
         return unique_id.key().id()
 
 class UsageCounter(db.Model):
-    """Counters count the historical statistics for a given repository.
-    To see how this is used, check out admin_stastics.py.
-    Unlike the Counter class, UsageCounter object increment when
+    """Counters which count the historical statistics for each repository.
+    To see how this is used, check out admin_statistics.py.
+    Unlike the Counter class, UsageCounter object increments when
     a new record or a new note is created, which means, the UsageCounter
     will not decrement when a record/note is expired/deleted."""
 
     # repo stored as a seperate property so it can be indexed and queried.
     repo = db.StringProperty(required=True)
-    # person_counter increment when a new person record is created.
+    # Total number of person records created so far,
+    # including deleted/expired ones.
     person_counter = db.IntegerProperty(default=0, required=True)
-    # note_counter increment when a new note is added in a person's record.
+    # Total number of notes created so far, including deleted/expired ones.
     note_counter = db.IntegerProperty(default=0, required=True)
 
     @staticmethod
-    def get(repo):
-        """Gets the entity with a given repository."""
-        return UsageCounter.get_by_key_name('%s' % repo)
-
-    @staticmethod
-    def add_person(count, amount):
-        """Do the calculation to increment the person_counter
-        This should be called in increment_person_counter method"""
-        count.person_counter += amount
-        count.put()
+    @db.transactional
+    def create(repo, person_counter=0, note_counter=0):
```

Comment:
Is there any place which specifies non-default person_counter and note_counter for this method? Otherwise you can remove them for now.

This would be good to read:
https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it